tests/length.normalization.test.ts

```diff
@@ -0,0 +1,30 @@
+import BM25 from "../BM25";
+
+describe("Length normalization", () => {
+    test("with b>0, same TF, shorter scores higher", () => {
+        const shortDoc = "x y y y"; // len=4
+        const longDoc  = "x " + "y ".repeat(99).trim(); //len=100
+        const q = ["x"];
+
+        const scores = BM25([shortDoc, longDoc], q, { k1: 1.2, b: 0.75 }) as number[];
+        expect(scores[0]).toBeGreaterThan(scores[1]); // test: shorter > longer
+    })
+
+    test("with b=0, length normalization off = equal score when TF equal", () => {
+        const shortDoc = "x y y y"; //tf=1
+        const longDoc = "x " + "y ".repeat(99).trim(); //tf=1
+        const q = ["x"];
+
+        const scores = BM25([shortDoc, longDoc], q, { k1: 1.2, b: 0 }) as number[];
+        expect(scores[0]).toBeCloseTo(scores[1],10);
```

Comment:
while this is a good use of jest to bypass floating point precision issues with JS, leave a comment stating you are expecting the values to be equal at to 10 decimal places.